src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/CommonDialog.cs

```diff
@@ -247,6 +247,7 @@ protected virtual IntPtr HookProc(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lP
             return IntPtr.Zero;
         }
 
+        /// <summary>
```

Comment:
An extra summary tag

Reply:
Fixed in https://github.com/dotnet/wpf/pull/7916/commits/543c8098a03629f522d8d86a9666b673a0e0a045